frontend/src/screens/Authentication/constants.tsx

```diff
@@ -0,0 +1,7 @@
+export const API_BASE_URL = 'http://localhost:5000';
```

Comment:
Get rid of the URL

Reply:
I am not sure I can remove this URL, cause it`s used in link`s href which leads to the backend authorization and href doesn't work with proxy